### ISSUE TITLE
One-Click RDP uses RedirPort

### ISSUE THREAD

**Issue by Sleepw4lker** [Ylianst/MeshCentral#912]
![Untitled](https://user-images.githubusercontent.com/1654791/73847803-46626780-4827-11ea-8e7d-8b8ff1a34cab.png)
Hi all,

first of all, MeshCentral is awesome, the more i used it, the more i love it.

So I played around with the routing capabilities and one-click RDP.

I notices that when Clicking the RDP Link of a Host, I get redirected to the Port configured with the RedirPort directive.

According to the Documentation: This is the port for redirecting traffic in the web server. When the server is configured with HTTPS, users that uses HTTP will be redirected to HTTPS. Port 80 is the default port. So, redirection will happen from port 80 to port 443.

I set RedirPort to 8082 as I have an Apache already running on Port 80 and 443 and use this as reverse Proxy. 8082 is not allowed via firewall and thus the connection fails. If I change the Port in the URL to 443, everything work as expected.

So IMHO it should not be necessary to use RedirPort?

My config for reference
```
{
  "settings": {
    "cert": "my.tomain.tld",
    "Port": 8081,
    "RedirPort": 8082,
    "MpsPort": 4433,
    "MpsTlsOffload": false,
    "AliasPort": 443,
    "AgentPong": 300,
    "TLSOffload": "127.0.0.1",
    "Minify": 1,
    "Wanonly": true,
    "ExactPorts": true
  },
  "domains": {
    "": {
        "newAccounts" : 0,
        "certUrl": "https://my.tomain.tld"
      }
    }
  }
}
```

**Comment by Ylianst** [comment]
Good catch, I will fix this. The HTTP (redir) port has to be used when the server does not have a real, trusted CA signed certificate because there is no way to say "I agree" to an invalid certificate with ClickOnce. However, if a real cert is used, HTTPS port must be used for this.

Working on fixing this...

**Comment by Ylianst** [comment]
Published MeshCentral v0.4.8-z with a fix for this. It will use HTTPS port unless an un-trusted TLS certificate is used.

**Comment by Sleepw4lker** [comment]
Confirmed working. Still, the App has several issues with SmartScreen Filter. Would perhaps a Code-Signature Certificate and automatic signature of the Apps help here?
